pkg/utils/resources.go

```diff
@@ -0,0 +1,75 @@
+package utils
+
+// Inspired from SNR - https://github.com/medik8s/self-node-remediation/blob/main/controllers/selfnoderemediation_controller.go#L283-L346
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	loggerResouces = ctrl.Log.WithName("resource")
+)
+
+func DeleteResources(r client.Client, nodeName string) error {
+	//fence
```

Comment:
The reboot is the fencing.
This is just improving rescheduling time.

Reply:
- This is just improving rescheduling time.
What about stateful pods (pods with PVC)? IIUC these kind of pods won't move/evict to another node